suite/src/__tests__/fast/composedb-model-sync.test.ts

```diff
@@ -77,9 +76,6 @@ describe('Sync Model and ModelInstanceDocument using ComposeDB GraphQL API', ()
     const documentId = responseObject?.data?.createBasicSchema?.document?.id
     expect(documentId).toBeDefined()
 
-    // Wait for the document to sync across nodes
-    await delay(nodeSyncDelay)
```

Comment:
Now that I see this line here, Could you explain why using the existing `waitForCondition` on the node we were expecting this to be synced on was not possible here? Like we do in this other [test](https://github.com/3box/ceramic-tests/blob/94c8c0bf45dcdf08c21cdd9b09d5723042baae84/suite/src/__tests__/slow/ceramic-ceramic-integration.test.ts#L51)

Reply:
For this test we only wanted to use GraphQL to connect with the nodes.

Reply:
The way I see it is that the `composedb2` instance is using the `ceramicInstance2` node so in the end its checking node events, node2 will try to get some info wont find it within itself and will go to the network to see who has it, in this case node1 has it, and you already have the documentId which is the streamId, my understanding is that after calling `const queryResponse = await composeClient2.executeQuery(` we add the wait for event waiting for the state to match that we loaded the event. I would like to have @stbrody's input here, since I don't really have a experience with this repo so much so he might have a better point of view regarding this.

Reply:
the queryResponse will not be defined if node2 does not know about the stream yet